unit_of_time/__init__.py

```
@@ -243,16 +313,24 @@ class Day(TimeunitKind):
     kind_int = 9
     formatter = "%Y-%m-%d"
 
+    @classmethod
+    def get_index_for_date(cls, dt):
+        return (dt - date.min).days
+
+    @classmethod
+    def get_date_from_index(cls, idx):
+        return date.min + timedelta(days=idx)
+
```

Ruff (0.13.3)

Unused class method argument: `cur`

(ARG003)